include/swift/AST/DiagnosticsParse.def

```diff
@@ -1812,6 +1818,9 @@ ERROR(avail_query_version_comparison_not_needed,
 ERROR(availability_query_wildcard_required, none,
       "must handle potential future platforms with '*'", ())
 
+ERROR(unavailability_query_wildcard_not_required, none,
+      "platform wildcard '*' is always implicit in '#unavailable'", ())
```

Comment:
Nit/Question: I just realized, that other diagnostics are not putting `#available` in quotes, so should we remove the quotes here?